docs/UserGuide.md

```diff
@@ -509,53 +532,55 @@ Example:
+* Information that you do not specify will be ordered last and follow the default order. (Tags > Phone Number > Email > Address)
 
 Example:
 
-* `customise order a/ e/ p/` The application will show address first, followed by email, phone number, then tags.
-* `customise order a/` The application will show address first. The rest of the information will follow the default order. Therefore, address will be followed by tags, phone number and then email.
+* `customise order a/ e/ p/` SoConnect will show address first, followed by email, phone number, then tags.
+* `customise order a/` SoConnect will show address first. The rest of the information will follow the default order. Therefore, address will be followed by tags, phone number and then email.
 
 <br>
 
 #### 7.5.2. Hiding contact details: `customise hide`
 
-Hide certain information of all contacts displayed.
+You can hide certain information of all contacts displayed.
 
 Format: `customise hide [t/] [p/] [e/] [a/]`
 
-* Information that can be hidden: Tags, Phone Number, Email, Address.
-* After using the command, the information specified is hidden.
-* If the information specified is already hidden, it will stay hidden.
+* You can hide the following information: Tags, Phone Number, Email, Address.
+* After you use this command, the information specified is hidden.
+* If the information that you specified is already hidden, it will stay hidden.
 
 Example:
-* `customise hide e/` The application no longer shows emails in the list of contacts.
-* `customise hide p/ t/` The application no longer shows phone numbers and tags in the list of contacts.
+* `customise hide e/` SoConnect no longer shows emails in the list of contacts.
+* `customise hide p/ t/` SoConnect no longer shows phone numbers and tags in the list of contacts.
 
 <br>
 
 #### 7.5.3. Showing contact details: `customise show`
 
-Show certain information of all contacts displayed.
+You can show certain information of all contacts displayed.
 
 Format: `customise show [t/] [p/] [e/] [a/]`
 
-* Information that can be changed from being hidden to being shown: Tags, Phone Number, Email, Address.
-* After using the command, the information specified is shown.
-* If the information specified is already shown, it will stay shown.
+* You can show the following information: Tags, Phone Number, Email, Address.
+* After you use this command, the information specified is shown.
+* If the information that you specified is already shown, it will stay shown.
 * `Coming soon in v1.5`, we will include `customise show all`, a shortcut to show all information.
 
 Example:
-* `customise show a/` The application now shows addresses in the list of contacts.
-* `customise show p/ t/` The application now shows phone numbers and tags in the list of contacts.
+* `customise show a/` SoConnect now shows addresses in the list of contacts.
+* `customise show p/ t/` SoConnect now shows phone numbers and tags in the list of contacts.
 
 <br>
 
```

Comment:
Perhaps can include note about upcoming commands of customisation for todo, to make this more general and not out of place.

Reply:
Can do after merging

Reply:
I added said note in the description for the customisation section